README.markdown

```diff
@@ -1,4 +1,142 @@
-# Common Lisp maintainers - and how to support them
+# Common Lisp maintainers - the ones we can support
 
 Sorted alphabetically by github handle. The authors' projects list is not exhaustive.
 
+In advance, thank you!
+
+
+## ambrevar - Pierre Neidhart
+
+- github: https://github.com/ambrevar
+- website: https://ambrevar.xyz/
+- donate:
+  - liberapay: https://liberapay.com/Ambrevar/donate
+- projects:
+  - Nyxt browser: https://github.com/atlas-engineer/nyxt
+
+## cxxxr
+
+- github: https://github.com/cxxxr/
+- donate:
+  - opencollective: https://opencollective.com/lem
+- projects:
+  - Lem: https://github.com/cxxxr/lem/
+  - cl-lsp: https://github.com/cxxxr/cl-lsp
+  - valtan: https://github.com/cxxxr/valtan
+
+## froggey
+
+- github: https://github.com/froggey/
+- donate:
+  - github sponsors: https://github.com/sponsors/froggey
+  - patreon: https://patreon.com/froggey
+- projects:
+  - Mezzano: https://github.com/froggey/Mezzano
+
+## hexstreamsoft - Jean-Philippe Paradis
+
+- github: https://github.com/Hexstream
+- website: https://www.hexstreamsoft.com
+- donate:
+  - github sponsors: https://github.com/sponsors/Hexstream
+- projects:
+  - https://www.hexstreamsoft.com/libraries/
+
+## jackdaniel - Daniel Kochmański
+
+- github: https://github.com/dkochmanski
+  - gitlab: https://gitlab.com/jackdaniel
+- website: https://github.com/McCLIM/McCLIM/
+- mastodon: https://mastodon.social/@jackdaniel
+- donate:
+  - patreon: https://www.patreon.com/jackdaniel%5Fkochmanski
+- projects:
+  - ECL: https://gitlab.common-lisp.net/ecl/ecl
+  - McCLIM: https://github.com/McCLIM/McCLIM/
+  - charming-clim: https://github.com/dkochmanski/charming-clim
+
+## jmercouris - John Mercouris
+
+- github: https://github.com/jmercouris
+- website: http://john.mercouris.online/
+- donate:
+  - patreon: https://www.patreon.com/nyxt
+- projects:
+  - Nyxt browser: https://github.com/atlas-engineer/nyxt
+
+## mfiano - Michael Fiano
+
+- github: https://github.com/mfiano
+- donate: https://github.com/sponsors/mfiano
+- projects:
+  - Virality Engine: https://github.com/bufferswap/ViralityEngine
+  - Pyx: https://github.com/mfiano/pyx
+  - pngload: https://github.com/bufferswap/pngload
+
+## phoe - Michał "phoe" Herda
+
+- github: https://github.com/phoe
+- website: https://phoe.github.io/
+- mastodon: https://functional.cafe/@phoe/
+- donate:
+  - github sponsors: https://functional.cafe/@phoe/
```

Comment:
Should be https://github.com/sponsors/phoe

Reply:
corrected. I had problems with my copy&paste this day.